### PR TITLE
Add Turkish translations

### PR DESCRIPTION
**Pull request by cem256**
Thank you for creating this amazing app. I wanted to contribute by translating to Turkish.

### CODE REVIEW

**Comment by Myzel394** [comment]
@cem256 thank you very much, this will help Locus a lot ❤️  I will take a look at your PR in a second, I first need to push a new release to fix the black screen from #28

File: l10n.yaml
Comment (Myzel394) [inline l10n.yaml:1]: Thanks for adding this file, I completely forgot to add it!

**Comment by Myzel394** [comment]
@cem256 I just realised you even translated the honorable mentions too! Thank you very much for your effort :)

**Comment by cem256** [comment]
You're most welcome! ❤️

**Comment by Myzel394** [comment]
@cem256 can you tell me what "Welcome, Turkish friends" means in Turkish? I would like to use this as the title for the next version (v0.4.1 probably) :D

**Comment by cem256** [comment]
Sure, you can use "Hoş geldiniz, Türk dostlarım"

**Pull request merged**